lib/hanami/app.rb

```diff
@@ -63,7 +63,7 @@ def mount(configuration)
     # @since 0.9.0
     # @api private
     def middleware(environment)
-      builder.use Hanami::CommonLogger, Hanami.logger
+      builder.use Hanami::CommonLogger, Hanami.logger if Hanami.logger
```

Comment:
`unless Hanami.logger.nil?`

If we introduce a _null object_ this test will still pass because of Ruby's truthiness:

```ruby
class NullLogger
  def nil?
    true
  end
end

Hanami.logger = NullLogger.new

if Hanami.logger
  # it enters this block, but it's unwanted
end

unless Hanami.logger.nil?
  # it doesn't enter this block as expected
end
```